tests/graph/operators/test_conv2d.py

```diff
@@ -38,23 +47,24 @@ def numpy_conv2d(data: np.ndarray, weight: np.ndarray, padding: List[int], strid
     return output
 
 
+@pytest.mark.parametrize("hidet_op", [ops.conv2d, ops.conv2d_gemm])
 @pytest.mark.parametrize(
-    "n, c, h, w, oc, kx, ky, padding, stride",
+    "n, c, h, w, oc, kx, ky",
     [
-        [1, 3, 32, 32, 12, 3, 3, [1, 1, 1, 1], [1, 1]],  # kernel 3, stride 1
-        [2, 3, 32, 32, 12, 3, 3, [1, 1, 1, 1], [1, 1]],  # kernel 3, stride 1, batch size 2
-        [1, 3, 32, 32, 12, 3, 3, [0, 0, 0, 0], [2, 2]],  # kernel 3, stride 2
-        [1, 3, 32, 32, 12, 1, 1, [1, 1, 1, 1], [1, 1]],  # kernel 1, stride 1
-        [1, 3, 32, 32, 12, 1, 1, [0, 0, 0, 0], [2, 2]],  # kernel 1, stride 2
-        [1, 3, 32, 32, 12, 7, 7, [3, 3, 3, 3], [2, 2]],  # kernel 7, stride 2
+        [1, 3, 32, 32, 12, 3, 3],  # kernel 3,
+        [2, 3, 32, 32, 12, 7, 7],  # kernel 7, batch size 2
+        [1, 3, 32, 32, 12, 1, 1],  # kernel 1,
     ],
 )
-def test_conv2d(n, c, h, w, oc, kx, ky, padding, stride):
+@pytest.mark.parametrize("padding", [[0, 0, 0, 0], [1, 1, 1, 1], [2, 2, 2, 2]])
+@pytest.mark.parametrize("stride", [[1, 1], [3, 4]])
```

Comment:
```suggestion
@pytest.mark.parametrize("stride", [[1, 1], [2, 2], [3, 4]])
```
`stride=(2, 2)` is also commonly used.